cmd/mayactl/app/command/volume.go

```diff
@@ -163,12 +163,15 @@ func NewVolumeInfo(URL string, volname string, namespace string) (volInfo *Volum
 		if resp.StatusCode == 500 {
 			fmt.Printf("Volume: %s not found at namespace: %q\n", volname, namespace)
```

Comment:
Can you check with @mahebbar how to work this error. Should be different from 404.

Reply:
Liked ur message so updated that